### PR TITLE
remove old help widget

### PR DESCRIPTION
**Pull request by Schparky**
With the new one just added, this old one needs to go.

### CODE REVIEW

File: src/data/analytics.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ export const createRequestByFab    = () => trackMenuEvent('create request', 'fab
 export const createRequestByButton = () => trackMenuEvent('create request', 'button')
 export const createRequestByTile   = () => trackMenuEvent('create request', 'tile')
 export const clickedLogo           = () => trackMenuEvent('logo clicked')
-export const clickedHelp           = () => trackMenuEvent('help')
```
Comment (forevermatt) [inline src/data/analytics.js:50]: Do we have a way to still collect this metric with the new help widget?

Comment (Schparky) [reply src/data/analytics.js:50]: I looked for a way, and the only answer I could come up with would be to create our own fab which opens the widget. I wasn't prepared to put that much work into it right now.

Comment (hobbitronics) [reply src/data/analytics.js:50]: `<Help on:mouseup={clickedHelp} />` wont work then?

Comment (hobbitronics) [reply src/data/analytics.js:50]: I see what you are saying now. You would need an element to fire the event.

**Review by hobbitronics — APPROVED** [review]

**Review by forevermatt — APPROVED** [review]

**Pull request merged**